src/queries/index.js

```diff
@@ -191,62 +191,25 @@ export const addUserToEvent = gql`
 `;
 
 export const removeUserFromEvent = gql`
-  mutation destroyUserEvent($input: DestroyUserEventInput!) {
-    removeUserFromEvent(input: $input) {
-      userEvent {
-        userId
-        eventId
+  mutation deleteUserEvent($input: DeleteUserEventInput!) {
+    deleteUserEvent(input: $input) {
+      event {
+        attendees{
+          id
+          username
+        }
       }
     }
   }
 `;
 
-// export const removeUserFromEvent = gql`
-//   mutation removeUserFromEvent($userId: ID!, $eventId: ID!) {
-//     removeUserFromEvent(userId: $userId, eventId: $eventId) {
-//       event {
-//         id
-//         date
-//         address
-//         state
-//         city
-//         zip
-//         title
-//         cancelled
-//         description
-//         hostId
-//         game
-//         gameType
-//         playerCount
-//         attendees {
-//           id
-//           username
-//         }
-//       }
-//     }
-//   }
-// `;
-
 export const cancelEvent = gql`
-  mutation cancelEvent($eventId: ID!) {
-    cancelEvent(id: $eventId, attributes: { cancelled: true }) {
+  mutation cancelEvent($input: CancelEventInput!) {
+    cancelEvent(input: $input) {
       event {
```

Comment:
So the refetching uses input(formerly known as put or patch) to update the information? Very cool!